site/content/docs/main/config/tls-termination.md

```diff
@@ -13,7 +13,12 @@ Contour also follows a "secure first" approach.
 When TLS is enabled for a virtual host, any request to the insecure port is redirected to the secure interface with a 301 redirect.
 Specific routes can be configured to override this behavior and handle insecure requests by enabling the `spec.routes.permitInsecure` parameter on a Route.
 
-The TLS secret must contain keys named tls.crt and tls.key that contain the certificate and private key to use for TLS, e.g.:
+The TLS secret must:
+- be a Secret of type `kubernetes.io/tls`. This means that it must contain keys named `tls.crt` and `tls.key` that contain the certificate and private key to use for TLS, in PEM format.
+
+The TLS secret may also:
+- add any additional certificates required for validation into the `tls.crt` PEM bundle. If this is the case, the serving certificate must be the first certificate in the bundle.
```

Comment:
Shouldn't the first bullet be removed? The additional certs in `tls.crt` should be **only** the chain CA certs which are there to support the peer when it is validating of our serving cert. Sending other certs would be redundant.

Reply:
Yep, I missed this, thanks.